cluster-provision/gocli/docker/docker.go

```diff
@@ -229,6 +231,13 @@ func NewCleanupHandler(cli *client.Client, cleanupChan chan error, errWriter io.
 								io.Copy(os.Stderr, reader)
 							}
 						}
+						err := cli.ContainerRemove(ctx, c, types.ContainerRemoveOptions{Force: true})
+						if err != nil {
+							fmt.Fprintf(errWriter, "%v\n", err)
```

Comment:
since we are going to retry them, and expect that it will fail, maybe don't print it here, just if the retry fails ?

style nit: (not mandatory)
```
if err := ...; err != nil {
}
```